examples/kafka/cifar10/cifar10_kafka.ipynb

```diff
@@ -0,0 +1,326 @@
+{
```

Comment:
<p>Would be good to expand the contents of the file with %writefile (similar to the example notebooks) so its possible to see whats inside even without having to run the cells</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/seldon-core/pull/1919/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
yes